platformio.ini

```diff
@@ -805,3 +806,13 @@ lib_deps =
   ${esp32.lib_deps}
   TFT_eSPI @ ^2.3.70
 board_build.partitions = ${esp32.default_partitions}
+
+# ------------------------------------------------------------------------------
```

Comment:
Actually one or two headlines should be enough ;-)

Reply:
Just saw that the previous entry also uses a FAT headline. So OK for me if you do the same.